test/controllers/declaration/SummaryControllerSpec.scala

```diff
@@ -133,6 +136,13 @@ class SummaryControllerSpec extends ControllerWithoutFormSpec with ErrorHandlerM
 
         status(result) must be(BAD_REQUEST)
       }
+
```

Comment:
Need to add tests that exercise the code that handles the lrn is a duplicate condition.

Reply:
I have tried to add another test that stubs `LrnValidator.hasBeenSubmittedInThePast48Hours` to emulate a duplicated lrn being found.

However the test seems to ignore the stub and I haven't been able to figure out why - any ideas?